pixeltable/env.py

```diff
@@ -251,6 +251,7 @@ def _create_nos_client(self) -> None:
             func.FunctionRegistry.get().register_module(mod)
 
     def _create_openai_client(self) -> None:
+        assert self._config is not None
```

Comment:
Maybe we should run set_up() in init. Or initialize to something non-None. It doesn't really make sense for these things to be None.